testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jaxrs/subresource/SubResourceTestCase.java

```diff
@@ -35,11 +30,15 @@
 import org.jboss.shrinkwrap.api.Archive;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.asset.EmptyAsset;
-import org.jboss.shrinkwrap.api.asset.StringAsset;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import java.net.URL;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
@jasondlee I suspect you have an IDE set to reorder imports according to a style that isn't the WF code style.

Reply:
I did, but I fixed that, but I guess those IDEA settings are per-project. I fixed these, and I guess I'll figure out how to make that default everywhere. :)